code/src/Core/PostActions/Catalog/Merge/PostActionFormatter.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Initially this not intended but it has occurred because Visual Studio when doing a renaming has changed the name of PostactionFormatter to PostActionFormatter. I can rollback this change, but I think the new name is the right one.